apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx

```diff
@@ -147,29 +149,40 @@ export const EditMultiFactorAuthenticator: FunctionComponent<EditMultiFactorAuth
                 if (error.response && error.response.data && error.response.data.description) {
                     dispatch(addAlert({
                         description: t("console:develop.features.authenticationProvider" +
-                            ".notifications.updateEmailOTPAuthenticator.error.description",
+                            ".notifications." + i18nKeyForMFAuthenticator + ".error.description",
                         { description: error.response.data.description }),
                         level: AlertLevels.ERROR,
                         message: t("console:develop.features.authenticationProvider" +
-                            ".notifications.updateEmailOTPAuthenticator.error.message")
+                            ".notifications." + i18nKeyForMFAuthenticator + ".error.message")
                     }));
 
                     return;
                 }
 
                 dispatch(addAlert({
-                    description: t("console:develop.features.authenticationProvider.notifications" +
-                        ".updateEmailOTPAuthenticator.genericError.description"),
+                    description: t("console:develop.features.authenticationProvider.notifications." +
+                        i18nKeyForMFAuthenticator + ".genericError.description"),
                     level: AlertLevels.ERROR,
-                    message: t("console:develop.features.authenticationProvider.notifications" +
-                        ".updateEmailOTPAuthenticator.genericError.message")
+                    message: t("console:develop.features.authenticationProvider.notifications." +
+                        i18nKeyForMFAuthenticator + ".genericError.message")
                 }));
             })
             .finally(() => {
                 setIsSubmitting(false);
             });
     };
 
+    const getI18nKeyForMultiFactorAuthenticator =
+        (authenticator: MultiFactorAuthenticatorInterface | AuthenticatorInterface) => {
+            if(authenticator.id === IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR_ID){
```

Comment:
```suggestion
            if (authenticator.id === IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR_ID) {
```

Reply:
Resolved in ea21716da10d86b7eb72ee5acfbe706711021e98